modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+After the AWS account user who initially created the DNS domain and the IAM roles, the VPC-owning AWS account user must create a private hosted zone and update the trust policy on the IAM role that was created for sharing the VPC.
+
+.Procedure
+
+. The AWS account user who owns the VPC must update the VPC sharing IAM role and add the Installer and Ingress Operator roles to the principal section of the trust policy.
++
+[source,terminal]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+	  "Sid": "Statement1",
+	  "Effect": "Allow",
+	  "Principal": {
+	  	"AWS": [
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/ManagedOpenShift-ingress-operator-cloud-credentials",
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/ManagedOpenShift-Installer-Role"
+        ]            
+	  },
+	  "Action": "sts:AssumeRole"
+	}
+  ]
+}
+----
+. After updating the trust policy, the VPC-owning AWS account user creates a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. The hosted spaces name is `<cluster-name>.<dns_domain>`. The private hosted zone needs to be associated with the created VPC.
```

Comment:
```suggestion
. After updating the trust policy, the *VPC-owning AWS account* user creates a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. In the hosted zone configuration, the domain name will be `<cluster-name>.<dns_domain>`. 
. After private hosted zone creation, it will need to be associated with the intended VPC.
```